src/preset_cli/cli/superset/sync/dbt/exposures.py

```diff
@@ -110,8 +120,21 @@ def sync_exposures(  # pylint: disable=too-many-locals
     for chart_id in charts_ids:
         chart = client.get_chart(chart_id)
         first_owner = chart["owners"][0]
+
+        asset_title = re.sub(
+            r"[^a-zA-Z0-9 _]",
+            "",
+            chart["slice_name"],
+        )  # remove unsupported characters
+        asset_title = re.sub(
+            r" ",
+            "_",
+            asset_title,
+        )  # replace blank spaces with underscores
```

Comment:
`\W` is equivalent to `[^a-zA-Z0-9_]` (no space, though), so if you replace spaces with underscores first you can simplify this:

```suggestion
        asset_title = re.sub(" ", "_", chart["slice_name"])
        asset_title = re.sub("\W", "", asset_title)
```

Reply:
ohhhhh, didn't know that! Thanks for sharing!